lib/std/os/windows/user32.zig

```diff
@@ -52,13 +86,21 @@ pub const WM_SYSKEYDOWN = 0x0104;
 pub const WM_SYSKEYUP = 0x0105;
 pub const WM_SYSCHAR = 0x0106;
 pub const WM_SYSDEADCHAR = 0x0107;
-pub const WM_UNICHAR = 0x0109;
-pub const WM_KEYLAST = 0x0109;
```

Comment:
I don't think you want to include most of the `*LAST` definitions: they merely state that this is the next free enum member, and it will likely change in future

Reply:
The keys are a subset of the `WM_` constants, so knowing what the last key or mouse event is specifically might still be useful.